src/components/ArticleListView.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ArticlePreview from 'components/ArticlePreview';
+
+export default class ArticleListView extends React.Component {
+  getInitialState() {
+    //getState()
+  }
+
+  componentDidMount() {
```

Comment:
This can either be a ControllerView, in which case you can do a falcor fetch or you can again make it pure and it will not need to interact with the store at all.